package.json

```diff
@@ -180,7 +180,7 @@
     "abstract-logging": "^2.0.1",
     "avvio": "^8.1.0",
     "fastify-error": "^1.0.0",
-    "find-my-way": "^5.1.0",
+    "find-my-way": "^5.3.0",
```

Comment:
This is unnecessary.

Reply:
Should I revoke or you can skip when merge?

Reply:
No need to change this at this point